spider/viewers/viewer_refinement.py

```diff
@@ -186,21 +186,17 @@ def _showVolumesChimera(self):
         """ Create a chimera script to visualize selected volumes. """
         volumes = self.getVolumeNames()
 
-        if len(volumes) > 1:
-            cmdFile = self._getFinalPath('chimera_volumes.cxc')
-            f = open(cmdFile, 'w+')
-            for vol in volumes:
-                # We assume that the chimera script will be generated
-                # at the same folder than spider volumes
-                localVol = os.path.basename(vol)
-                if os.path.exists(vol):
-                    f.write("open %s format spider\n" % localVol)
-            f.write('tile\n')
-            f.close()
-            view = ChimeraView(cmdFile)
-        else:
-            view = ChimeraClientView(volumes[0])
```

Comment:
Thanks, @rmarabini . Is ChimeraClientView then gone completely?

Reply:
Yes